server/chat-plugins/info.js

```diff
@@ -1520,7 +1520,9 @@ const commands = {
 	},
 
 	'!calc': true,
+	bsscalc: 'calc',
 	calculator: 'calc',
+	cantsaycalc: 'calc',
```

Comment:
I'm not sure about 'cantsaycalc'. 'calc' isn't aliased to 'honkocalc' or anything?

Reply:
it's fairly frequently referred to as 'cantsaycalc' in the battle spot room. @cantsay can verify if needed

Reply:
I'm personally not a fan of embedding a username in the command and there's already credit attributed when the link is displayed. If Zarel or someone else wants to merge then I wouldn't complain, but this is what is blocking me FWIW.

Reply:
Tbf I’d imagine cantsay was the one who first called it cantsay calc and it got more used because he is the bss leader..

Reply:
I just am not a huge fan of this kind of branding/credits. It personally comes across thirsty to me. I think projects should stand on their own, and be built/maintained by teams or organizations, not individuals who are likely to move on to other things. This is just my view though, so I can completely understand others feeling differently.

While we're on the topic of credits though, `(Courtesy of Honko)` should probably be adjusted to `(Courtesy of Honko &amp; Austin)`, but thats also my opinion. It bugs me that as developers we only think to acknowledge the original developers and not the people who do the thankless task of maintenance/stewardship.

Anyway, I apologize for dragging this thread of comments into a political direction or assigning any motive to the alias. I just wanted to make it clear that it is the sole reason I haven't merged (I'd happily merge this minus `/cansaycalc`  and then you can have someone else approve the alias as its own PR as followup)

Reply:
You'd be scandalized to know that we used to have commands ``/joim`` and ``/birkal`` (yeah, Birkal is another user).

Personally, I'd prefer it if we didn't have this sort of branding, but I'd have no issue if pragmatism won in the end (so long as the claimed ongoing usage is true).

Reply:
I mean, as I said, I wouldn't even be *that* scandalized if `/cantsaycalc` gets merged, but if its up to me (ie. like  the current decision as to whether or not *I'm* the one that merges it), I'm going to go with what I'm comfortable with.

Reply:
I'm generally fine with branding. I do agree changing the damagecalc credit to "courtesy of Honko & Austin".

We still use "GXE" which stands for "Glicko X-Act Estimate" – that's _two_ vanity names in one! I tried to rebrand with a slightly different "WCAO - Win Chance vs Average Opponent" but sadly it was fatally flawed.

For that matter, Elo and Glicko are also vanity names. I'm pretty sure the ship has sailed.

I do think it's kind of random/unfair who gets credited and who doesn't, but whatever. Like @Slayer95 said, pragmatism wins out. I think even worse is having a reputation for "hating fun".

Reply:
IMO a "fun" name is like a meme one or something ironic etc. And I'm all for those (within reason). 

In my ideal world we'd have metrics around command usage and could see how often `/cantsaycalc` gets used relative to just `/calc` or `/bsscalc`. If its truly popular enough to justify the branding then I'm fine. I can't argue with data. But of course, thats overkill to set up for this (and we'd only have the data *after* we merged).